test/e2e/specs/published-content/likes__comment.ts

```diff
@@ -30,56 +28,86 @@ describe( DataHelper.createSuiteTitle( 'Likes (Comment) ' ), function () {
 			accountName: 'simpleSitePersonalPlanUser',
 		},
 	] );
-
 	const comment = DataHelper.getRandomPhrase();
+	const postContent =
+		'The foolish man seeks happiness in the distance. The wise grows it under his feet.\n— James Oppenheim';
 	let page: Page;
-	let commentsComponent: CommentsComponent;
-	let editorPage: EditorPage;
+	let newPost: PostResponse;
+	let newComment: NewCommentResponse;
+	let restAPIClient: RestAPIClient;
+	let testAccount: TestAccount;
 
-	beforeAll( async () => {
+	beforeAll( async function () {
 		page = await browser.newPage();
-		editorPage = new EditorPage( page, { target: features.siteType } );
 
-		const testAccount = new TestAccount( accountName );
+		testAccount = new TestAccount( accountName );
+
+		restAPIClient = new RestAPIClient( testAccount.credentials );
+
+		newPost = await restAPIClient.createPost(
+			testAccount.credentials.testSites?.primary.id as number,
+			{
+				title: DataHelper.getTimestamp() as string,
+				content: postContent,
+			}
+		);
+
+		newComment = await restAPIClient.createComment(
+			testAccount.credentials.testSites?.primary.id as number,
+			newPost.ID,
+			comment
+		);
```

Comment:
The spec now uses API calls to set up the test post and comment.

I thought about using a locked and always-existing post and simply add a new comment to that, but I decided against that because there is no guarantee the "permanent" post will not be wiped out by an inadvertent headstart or theme change.

Reply:
After chatting with @fullofcaffeine in p1679099808423219/1678991483.901129-slack-CBTN58FTJ, an intermediate approach is to set up the test post programmatically (we already have many other specs that test the editor posting experience), but post comments and like/unlike using the UI.